roles/kubernetes/node/defaults/main.yml

```diff
@@ -11,3 +11,6 @@ kube_proxy_mode: iptables
 # kube_api_runtime_config:
 #   - extensions/v1beta1/daemonsets=true
 #   - extensions/v1beta1/deployments=true
+
+nginx_image_repo: nginx
+nginx_image_tag: alpine
```

Comment:
I'll freeze it to 1.11.4-alpine. The alpine backed image is smaller than the main one.